spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/build.gradle

```diff
@@ -59,6 +59,14 @@ configurations {
 dependencies {
 	compile gradleApi()
 	compile localGroovy()
+	compile "org.eclipse.aether:aether-api:${aetherVersion}"
```

Comment:
`api` makes sense since it's not shaded but I also need to pass the rest even though it should get shaded